app/_services/frameState.ts

```diff
@@ -126,7 +126,7 @@ export const getUpdatedFrameState = (
 			if (buttonValue === 'submit' && currentDeposit.min !== undefined) {
 				if (inputText && isNaN(Number(inputText))) {
 					updatedState.error = 'Value must be a number';
-				} else if (inputText && parseFloat(inputText) >= currentDeposit.min) {
+				} else if (inputText && Number(inputText) >= Number(currentDeposit.min)) {
```

Comment:
<img width="857" alt="Screenshot 2024-10-18 at 16 55 53" src="https://github.com/user-attachments/assets/d4fdec00-ac7c-414a-9e3c-3a18ca678b43">

check line 49 of this file, it says "greater than"